test/validators/transfer-bulk.ts

```diff
@@ -0,0 +1,137 @@
+      podResult3.podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    )).to.emit(ssvNetworkContract, 'BulkValidatorTransferred');
+  });
+
+  it('Bulk transfer validator with an invalid owner', async () => {
+    // Transfer validator with an invalid owner
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[5]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('ValidatorNotOwned');
+
+    // Transfer validator with an unowned validator
+    const account5Pod = await helpers.registerValidators(5, 1, '10000', helpers.DataGenerator.pod.new(), [GasGroup.REGISTER_VALIDATOR_NEW_STATE]);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, account5Pod.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length + 1).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('ValidatorNotOwned');
+
+    // Transfer with an invalid public key
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, helpers.DataGenerator.shares(0), ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length + 1).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('ValidatorNotOwned');
+  });
+
+  // NEED TO MAKE NEW GAS GROUP FOR BULK TRANSFER
+  it('Bulk transfer 10 validators', async () => {
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    ), [GasGroup.TRANSFER_VALIDATOR_EXISTED_POD]);
+  });
+
+  // NEED TO MAKE NEW GAS GROUP FOR BULK TRANSFER
+  it('Bulk transfer 10 validators to a pod with 7 operators', async () => {
+    // Register validator with 7 operators
+    const { podId } = await helpers.registerValidators(4, 1, '90000', [1, 2, 3, 4, 5, 6, 7]);
+
+    // Transfer validator to an existing pod
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    ), [GasGroup.TRANSFER_VALIDATOR_EXISTED_POD]);
+  });
```

Comment:
@lior-blox gas is above expected here (but im using gas group of "transfer" whereas this is bulk transfer)
we need to decide on some expected gas values for bulk transfer validators